spot_driver/test/conftest.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute LLC. See LICENSE file for more info.
```

Comment:
Having just sat down to write a test that needs to mock spot2 but not in this directory could some of this utility live in a place it can be imported into other tests?

Reply:
Also the copyright should be 2023-2024

Reply:
At the moment the utility lives in SpotWrapper. During last meeting I was suggesting to move it in its own package.